### DISCUSSION TITLE
KEY is not displayed

### DISCUSSION THREAD

**Discussion by henriquelucas** [rustdesk/rustdesk#2114] [General]
I installed Rust Desk Server on an Ubuntu 20.04.
All right, when I go on the Client side I add the address of my server Everything connects but I am not able to find the KEY.

https://i.ibb.co/hYC1W2H/SS.png

**Comment by pedjas** [comment]
You have to specify host and key in exe name. Only then they would be used. If you omit key in exe name, then it would connect to default public host. At least that is what I concluded.

Using self host is quite confusing.
